.travis_do.sh

```diff
@@ -34,60 +50,61 @@ download_sdk() {
 
 	# check again and fail here if the file is still bad
 	sha256sum -c ./sha256sums.small
-	echo "=== SDK is up-to-date"
+	echo_blue "=== SDK is up-to-date"
 }
 
 # test_package will run on the `script` step.
 # test_package call make download check for very new/modified package in it's
 # own clean sdk directory
 test_packages() {
 	# search for new or modified packages. PKGS will hold a list of package like 'admin/muninlite admin/monit ...'
-	PKGS=$(git diff --stat "$TRAVIS_COMMIT_RANGE" | grep Makefile | grep -v '/files/' | awk '{ print $1}' | awk -F'/Makefile' '{ print $1 }')
+	PKGS=$(git diff --name-only "$TRAVIS_COMMIT_RANGE" | grep 'Makefile$' | grep -v '/files/' | awk -F'/Makefile' '{ print $1 }')
 
 	if [ -z "$PKGS" ] ; then
-		echo "No new or modified packages found!" >&2
+		echo_blue "No new or modified packages found!" >&2
 		exit 0
 	fi
 
-	echo "=== Found new/modified packages:"
+	echo_blue "=== Found new/modified packages:"
 	for pkg in $PKGS ; do
 		echo "===+ $pkg"
 	done
 
+	echo_blue "=== Setting up SDK"
+	tmp_path=$(mktemp -d)
+	cd "$tmp_path"
+	tar Jxf "$SDK_HOME/$SDK.tar.xz" --strip=1
+
+	cat > feeds.conf <<EOF
```

Comment:
Please add a comment, why the script is using github feeds instead the officials.